packages/server/assets/server.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDBDCCAeygAwIBAgIJAOHcgvlzrH3IMA0GCSqGSIb3DQEBCwUAMBQxEjAQBgNV
+BAMMCWxvY2FsaG9zdDAeFw0yMDA3MDUyMTAyMDdaFw0yMDA4MDQyMTAyMDdaMBQx
+EjAQBgNVBAMMCWxvY2FsaG9zdDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoC
+ggEBAMomMbI/OMj0gwo+mcCuv8mrcZQ6h8eTdMUKFE6Gakrbcs9Oj1wavv1UO/03
+y1CnzrPJAci5/Y43NGD9Chc6hKu6I9RA467QuWFO7r1fRxCs2pdTI2u+I0e35Cv2
+Njvjo83W6SRVPY4/CM7g1ceW6OAs15whODhVuObonwDTQEP/WJWzqxVO21zIesdD
+fiZk8nQfE+itdkPm+1BZF5uxzH1BV1Bc3lueJPfwQ3VwWsrWtxw/Tt9QxXgunJX8
+L/Gs5YBlKWXzmsPcUHNP9rrRUx2tOITkf9MLWAWAeh/O6QPpGG5jyp/LhQ+yJD1q
+UqMT6B0i2VcuRGL6voePsQFHg5kCAwEAAaNZMFcwMwYDVR0RBCwwKoIoZWMyLTUy
+LTkwLTE5OC0zOC5jb21wdXRlLTEuYW1hem9uYXdzLmNvbTALBgNVHQ8EBAMCB4Aw
+EwYDVR0lBAwwCgYIKwYBBQUHAwEwDQYJKoZIhvcNAQELBQADggEBAELs1IhY1sto
+w5/XTpRQQGZo2aBpfhRvLPa5kBR/7BWHOS+iKJ73aGzijbaRoCoe1us2/AZeCNb4
+ZphMnrG6CYum11AeuSP+i4vGozx0tWYBjNycjKWbpACPJTqefYxFdA9z/8UCUe1d
+NOYfFES1D7p6mz7AzD7w3yOXdJeQ5htoHtxIHqbJVu0UPmLpg/YmxX3N1tD+fUPG
+2DppVqIWDwptZV0Zl13aojayAKMjrDhbJXcNB0wEmUm7iovzOzjKR/isDQKFVW5V
+gHP0bJIzVgN/AAKQhe4SjBcdzYjjyVjLTR/hDjPaivXZBwTSktNxexYFYPrj3AbW
+jGtEzdWrfyg=
+-----END CERTIFICATE-----
```

Comment:
Makes sense to store certificates aside

Reply:
Should put assets to gitignore than and add the generated certificates manually. Can we do that?